### PR TITLE
Updates to latest o-colors

### PR DESCRIPTION
**Pull request by AlbertoElias**

### CODE REVIEW

File: demos/basic.html [hunk 1]
```diff
@@ -1,7 +1,5 @@
 <!DOCTYPE html>
-<!--[if IE 7]>         <html class="o-useragent-ie7 o-hoverable-on"> <![endif]-->
-<!--[if IE 8]>         <html class="o-useragent-ie8 o-hoverable-on"> <![endif]-->
-<!--[if gt IE 8]><!--> <html class="o-hoverable-on"> <!--<![endif]-->
+<html class="o-hoverable-on ">
```
Comment (kaelig) [inline demos/basic.html]: Funny that there is a space at the end of the class value

**Comment by kaelig** [comment]
:+1:

File: demos/basic.html (hunk 1 above)
Comment (AlbertoElias) [inline demos/basic.html]: It doesn't affect in any way. I thought i wasn't a bad compromise. It can be removed, but it would make the template messier

Comment (kaelig) [inline demos/basic.html]: No worries, just found it funny

**Pull request merged**